OctoKit/Releases.swift

```diff
@@ -18,8 +18,8 @@ public struct Release: Codable {
     public let url: URL
     public let htmlURL: URL
     public let assetsURL: URL
-    public let tarballURL: URL
-    public let zipballURL: URL
+    public let tarballURL: URL?
+    public let zipballURL: URL?
```

Comment:
Can you elaborate why this change was necessary? It would be a breaking change.

Reply:
These properties return null and cause `DecodingError` when `draft` is true.
Tarball and zipball are not available on draft release.